cirq/ops/two_qubit_noise_channels.py

```diff
@@ -0,0 +1,1139 @@
+# Copyright 2019 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import cmath
+from typing import Sequence, Tuple
+
+import numpy as np
+
+import cirq
+
+from cirq import value, protocols
+
+from cirq._compat import proper_repr
+from cirq.ops import gate_features, eigen_gate, raw_types, pauli_gates, common_gates
+
+
+@cirq.value.value_equality
+class TwoQubitAsymmetricDepolarizingChannel(gate_features.TwoQubitGate):
```

Comment:
Move this into a file `cirq/contrib/noise_models/two_qubit_noise_models.py`. Also add a file that tests it in that `noise_models_directory`?